staff/pep-delbano/cv/styles.css

```diff
@@ -17,93 +25,221 @@
     "foot";
 }
 
-@media (min-width: 500px) { /*tablet size*/
-    .grid-container {
-        grid-template-columns: 1fr 1fr;
-        grid-template-areas: 
-        "header image"
-        "lefty right1"
-        "centered right2"    
-        "foot foot";
-    } 
-}
-
-
-@media (min-width: 750px) { /*desktop size*/
-    .grid-container {  
-        display: grid;
-        grid-template-columns: 1fr, 3fr, 2fr;
-        grid-auto-rows: minmax(100px, auto auto auto);
-        grid-gap: 15px;
-        grid-template-areas:
-        "header header image"
-        "lefty centered right1"
-        "lefty centered right2"    
-        "foot foot foot"; 
-    }
-}
-
+.image, .leftside, .rightside1, .rightside2, .inventions, .neverBuilt, .div__head {
+    border-radius: 5px;
+    min-width: 100px;
+    width: 90%;
+    padding: 10px;
+    margin-bottom: 10px;
+    box-shadow: 3px 3px 3px 1px hsl(42, 86%, 45%);
 
-.grid-container ul {
-    list-style: none;
-    margin: 0;
-    padding: 0;
+    background-color: #f7f4e3;
+    -webkit-transition: background-color 1500ms linear;
+    -ms-transition: background-color 1500ms linear;
+    transition: background-color 1500ms linear;
 }
 
 .header {
-    grid-area: header;
-    width: 100%;
-    background-color: hsl(183, 69%, 87%);
-    padding: 20px;
+    max-width: 150px;;
+    width: 90%;
 }
-
 .image {
     grid-area: image;
-    display: inline-block;
-    border-radius: 55%;
-    min-width: 100px;
+    width: 90%;
+    max-width: 120px;
+    margin-left: 30px;
+    border-radius: 50%;
+    box-shadow: 3px 3px 3px 1px hsl(42, 86%, 45%);   
 }
 
 .leftside {
     grid-area: lefty;
-    background-color: hsl(183, 69%, 87%);
-    padding: 10px;
-    width: 100%;
 }
 
 .rightside1 {
     grid-area: right1;
-    background-color: hsl(183, 69%, 87%);
-    padding: 10px;
-    width: 95%;
 }
 
 .rightside2 {
     grid-area: right2;
-    background-color: hsl(183, 69%, 87%);
-    padding: 10px;
-    width: 100%;
 }
 
 .inventions {
     grid-area: centered;
-    background-color: hsl(183, 69%, 87%);
-    padding: 10px;
-    width: 90%;
 }
 
 .neverBuilt {
     grid-area: foot;
-    background-color: hsl(183, 69%, 87%);
-    padding: 10px;
-    width: 100%;
 }
 
-h1 {
+
+.header_maintittle {
     font-size: 35px;
-    font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+    width: 90%;
+    font-family: 'Lobster', cursive;    
+    text-shadow: 1px 2px 3px white;
+    letter-spacing: 4px;
 }
 
-h2 {
+.div__head{
```

Comment:
`div__head` is NOT semantic, may you wanted to mean something like `container__head` or similar? AVOID using `div` as a prefix for a BEM selector, it's agains the principle of building concepts (blocks, objects)